packages/fast-usdc/test/cli/lp-commands.test.ts

```diff
@@ -0,0 +1,114 @@
+import { makeMarshal } from '@endo/marshal';
+import anyTest, { type TestFn } from 'ava';
+import { Command } from 'commander';
+import { flags } from '../../tools/cli-tools.js';
+import { mockStream } from '../../tools/mock-io.js';
+import { addLPCommands } from '../../src/cli/lp-commands.js';
+
+const makeTestContext = () => {
+  const program = new Command();
+  program.exitOverride();
+  const out = [] as string[];
+  const err = [] as string[];
+
+  const USDC = 'usdcbrand';
+  const FastLP = 'fastlpbrand';
```

Comment:
I'm surprised `markeMarshal` doesn't require these to be remotables. I usually have to do:

```suggestion
  const USDC = Far('usdcbrand');
  const FastLP = Far('fastlpbrand');
```

Reply:
There was no type error either way. However, now that we're using AmountMath and such, it does expect the brand to be remotable, so done.